ui/app/appointments/controllers/manage/appointmentsCreateController.js

```diff
@@ -541,5 +558,43 @@ angular.module('bahmni.appointments')
                 $state.go('^', $state.params, {reload: true});
             };
 
+            $scope.isUserManageOwnAppointmentPrivilegedOnly = function () {
```

Comment:
return (appointmentCommonService.isCurrentUserHasPrivilege(ownAppointmentPrivilege, currentUserPrivileges) && !appointmentCommonService.isCurrentUserHasPrivilege(manageAppointmentPrivilege, currentUserPrivileges));
This will return true when user has manage own appointments but not manage appointments privilege.

Reply:
yep, thats alright. I was requesting for a better name.

Reply:
is doesUserHaveManageOwnAppointmentPrivilegeOnly fine?

Reply:
"canManageOwnAppointmentOnly" ? its then a more  context centric, rather than asking a privilege level question.